_posts/2011-10-25-odata-service-validation-tool-update-new-feature-and-rules.html

```diff
@@ -20,7 +20,7 @@
 </ul>
 <p>I’d like to highlight the new crawling feature. Since we launched one of the requests that we heard was to be able to hierarchically validate a service starting from a service document. We had added the capability to do that in the engine in the last release. In this release, we added UI support for the feature.</p>
 <p>If you enter enter a URL to a service document and select the crawling checkbox, the validation engine will automatically validate the service doc, the metadata document (if available), the top feed in the service document and the top entry in the feed. In addition to those we also send a bad request to generate an OData error payload and validate that as well.</p>
-<p>As always we’d like to hear your feedback. Please check the new feature out and let us know what you think either on the <a href="https://www.odata.org/join-the-odata-discussion/">mailing list</a> or on the <a href="http://odatavalidator.codeplex.com/discussions">discussions page</a> on the Codeplex site.</p>
+<p>As always we’d like to hear your feedback. Please check the new feature out and let us know what you think either on the <a href="/join-the-odata-discussion/">mailing list</a> or on the <a href="http://odatavalidator.codeplex.com/discussions">discussions page</a> on the Codeplex site.</p>
```

Comment:
This link is broken http://odatavalidator.codeplex.com/discussions

Reply:
@KenitoInc Fixing dead links that point to external resources was beyond the scope of this pull request. In any event, I didn't have a valid alternative I could replace them with.